components/common/Button/DangerButton.tsx

```
@@ -0,0 +1,44 @@
+import { ReactNode } from "react";
+import { Button } from "./Button";
+import { FiTrash2 } from "react-icons/fi";
+import { LoadingIndicator } from "@/components/common/LoadingIndicator";
+
+interface DangerButtonProps {
+	onClick: (...args: any[]) => void;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)